source/alloy-service.ts

```
@@ -0,0 +1,72 @@
+import { Process } from "types";
+import { chalk, commander } from "../vendor/npm";
+import { lookupService } from "./service/utils";
+import Client from "./service/client";
+import Server from "./service/server";
+
+/**
+ * alloy-service.js
+ *
+ * Alloy service.
+ *
+ * @author Joel Ong (joelo@google.com)
+ * @copyright 2015 Notion Labs, Inc
+ */
+
+commander
+    .command("start [pathspec...]")
+    .description(`Starts Alloy service and optionally watches files
+                     given by [pathspec...].
+`)
+    .action(start);
+
+commander
+    .command("stop")
+    .description("Stops Alloy service.")
+    .action(stop);
+
+commander.parse(process.argv);
+
+// Output help if no command was provided.
+if (!process.argv.slice(2).length
+    || (process.argv[2] !== "start" && process.argv[2] !== "stop")) {
```

Unrecognized commands should throw an error instead of producing the help screen.